### ISSUE TITLE
Support using DATABASE_URL ENV var

### ISSUE THREAD

**Issue by Serdnad** [amberframework/micrate#49]
Would it be possible to use the DATABASE_URL environment variable, possibly as a second option if DB_URL is not set?

As far as I can tell, DATABASE_URL is more standard than DB_URL.

**Comment by pynixwang** [comment]
```
require "micrate"
require "pg"

Micrate::DB.connection_url = ENV["DATABASE_URL"]
Micrate::Cli.run
```

**Comment by pynixwang** [comment]
default should be `DATABASE_URL`

**Comment by drujensen** [comment]
This is fixed in v0.9.0
